contracts/interfaces/IP2PPairStaking.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.8.10;
+
+import "../dependencies/openzeppelin/contracts/IERC20.sol";
+
+interface IP2PPairStaking {
+    enum StakingType {
+        BAYCStaking,
+        MAYCStaking,
+        BAYCPairStaking,
+        MAYCPairStaking
+    }
+
+    struct ListingOrder {
+        StakingType stakingType;
+        address offerer;
+        address token;
+        uint32 tokenId;
+        uint32 share;
+        uint256 startTime;
```

Comment:
we can use smaller uint here. uint64 should be enough for timestamp

Reply:
ListingOrder is not stored on chain. so it's ok